packages/app-desktop/gui/NoteRevisionViewer.tsx

```diff
@@ -164,6 +177,7 @@ const NoteRevisionViewerComponent: React.FC<Props> = ({ themeId, noteId, onBack,
 	}
 
 	const restoreButtonTitle = _('Restore');
+	const deleteAllButtonTitle = _('Delete All');
```

Comment:
I would prefer something a bit clearer like "Delete history" or maybe "Delete all revisions". Could you change this please? But also everywhere where things are named "deleteAll", it should be "deleteHistory"